portfolio/src/main/webapp/style.css

```diff
@@ -63,4 +67,30 @@ img.ctr {
 p.ctr {
     display: block;
     margin: 0 auto;
+}
+
+#blog{
+    display:none;
+    
+}
+
+.blogtext{
+    float: left;
+    font-family: Raleway, sans-serif;
+    font-size: 20px;
+    display:inline;
+}
+
+.blogimg{
+    float:right;
```

Comment:
I think float and inline display do different things that aren't really compatible, or should't work nicely together.

If you open chrome developer extensions with ctrl+shift+i, you can look at the DOM of the page as chrome parses it, and look at individual elements to see which styles apply, and where in the stylesheet they're coming from. It will also let you check and uncheck specific styles to see how they affect the output, and show some styles as crossed out so you know they're being overridden elsewhere. You can also make temporary changes to particular styles to see how those would affect the output (though those changes aren't saved, you have to then add them to the stylesheet file to make them permanent).

Reply:
Right, they're conflicting, my bad. The flex container overrides these but I forgot to take them out. Thanks for suggesting looking at the DOM.